spaces/tests/test_unit_importer.py

```diff
@@ -92,7 +92,7 @@ def test_importer_creates_a_units_with_correct_values(self, mock_response):
 
         unit_too = Unit.objects.get(service_map_id=2)
         self.check_unit_values_vs_data_values(unit_too, data[1])
-        location_too = Location.objects.get(unit=2)
```

Comment:
have no idea why this passed the tests before and why this failed now

Reply:
now I know. The `Location.objects.get(unit=[integer])` where the integer refers to pk of the fk reference. TIL.